includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -209,8 +214,11 @@ class DesignSystemApiController extends WikiaApiController {
 
 	public function getFooter() {
 		$params = $this->getRequestParameters();
-
-		$footerModel = new DesignSystemGlobalFooterModel( $params[ 'wikiId' ], $params[ 'lang' ] );
+		$footerModel = new DesignSystemGlobalFooterModel(
+			$params[self::PARAM_PRODUCT],
+			$params[self::PARAM_ID],
+			$params[self::PARAM_LANG]
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)